<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lessonselectwindow</class>
 <widget class="QWidget" name="lessonselectwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: black;</string>
  </property>
  <widget class="QFrame" name="headerFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-bottom: 2px solid black;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="headerLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Old London</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>-1</number>
    </property>
    <property name="text">
     <string>Lesson Select</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: transparent;
background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>1191</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>24</number>
    </property>
    <item row="0" column="1">
     <widget class="QPushButton" name="lessonButton_2">
      <property name="minimumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>OldNewspaperTypes</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
      </property>
      <property name="text">
       <string>
Alphabet 1
e, t, a, n, m, i</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="lessonButton_10">
      <property name="minimumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>OldNewspaperTypes</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
      </property>
      <property name="text">
       <string>
The Paddle</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="lessonButton_6">
      <property name="minimumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>OldNewspaperTypes</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
      </property>
      <property name="text">
       <string>
Alphabet 5
z, v, y, q, p, x</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="lessonButton_9">
      <property name="minimumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>OldNewspaperTypes</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
      </property>
      <property name="text">
       <string>
Words</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="lessonButton_3">
      <property name="minimumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>OldNewspaperTypes</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
      </property>
      <property name="text">
       <string>
Alphabet 2 
k, r, s, o</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="lessonButton_4">
      <property name="minimumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>OldNewspaperTypes</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
      </property>
      <property name="text">
       <string>
Alphabet 3
 g, w, d, u</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="lessonButton_5">
      <property name="minimumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>OldNewspaperTypes</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
      </property>
      <property name="text">
       <string>
Alphabet 4
b, j, f, l, c, h</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="lessonButton_1">
      <property name="minimumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>OldNewspaperTypes</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
      </property>
      <property name="text">
       <string>
Morse Code
Basics</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="lessonButton_8">
      <property name="minimumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>OldNewspaperTypes</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
      </property>
      <property name="text">
       <string>
Numbers
Practice</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="lessonButton_7">
      <property name="minimumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>164</width>
        <height>164</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>OldNewspaperTypes</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
      </property>
      <property name="text">
       <string>
Numbers
Basics</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="instructionTextLabel">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>640</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OldNewspaperTypes</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SELECT a lesson to learn morse code!</string>
   </property>
  </widget>
  <widget class="QLabel" name="lessonCompleteLabel">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>640</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OldNewspaperTypes</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LESSONS COMPLETE:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lessonCompleteCounterLabel">
   <property name="geometry">
    <rect>
     <x>1161</x>
     <y>640</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OldNewspaperTypes</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
